webpack/assets/javascripts/react_app/redux/API/APISelectors.js

```diff
@@ -0,0 +1,14 @@
+import { STATUS } from '../../constants';
+
+export const selectAPI = store => store.API;
+
+export const selectAPIByKey = (store, key) => selectAPI(store)[key] || {};
+
+export const selectAPIProperty = (store, key, property) =>
+  selectAPIByKey(store, key)[property];
+
+export const selectAPIStatus = (store, key) =>
+  selectAPIProperty(store, key, 'status') || STATUS.PENDING;
+
+export const selectAPIError = (store, key) =>
+  selectAPIProperty(store, key, 'error') || null;
```

Comment:
`selectAPIProperty` feel unnecessary for me, would look better as:
```js
export const selectAPIStatus = (store, key) =>
  selectAPIByKey(store, key).status || STATUS.PENDING;

export const selectAPIError = (store, key) =>
  selectAPIByKey(store, key).error || null;
```

Reply:
If we won't fix the state structure, the property selector was used also in the components, though if we will get a convention we could use the results selector